edkrepo/commands/combo_command.py

```diff
@@ -47,13 +47,13 @@ def run_command(self, args, config):
             combo_list.append(manifest.general_config.current_combo)
         for combo in sorted(combo_list):
             if combo == manifest.general_config.current_combo:
-                print("* {}{}{}".format(Fore.GREEN, combo, Fore.RESET))
+                ui_functions.print_info_msg("current combo: " + combo, header=False)
             elif combo in combo_archive:
-                print("  {}{}{}{}".format(Fore.YELLOW, Style.BRIGHT, combo, Style.RESET_ALL))
+                ui_functions.print_info_msg("archived combo: " + combo, header=False)

```

Comment:
As we are trying to emulate the formatting of git branch lets retain the original formatting. While the color of this is not standard I think it will look better overall than if we add the extra string.